src/main/java/rx/observables/SwingObservable.java

```diff
@@ -173,7 +175,7 @@ public Boolean call(KeyEvent event) {
     /**
      * Creates an observable corresponding to item events.
      * 
-     * @param component
+     * @param itemSelectable
```

Comment:
Let's keep pull requests specific, and have separate PRs for small cleanups like this. (It's ok this time.)